test/e2e/cluster_extension_install_test.go

```diff
@@ -94,6 +94,18 @@ func TestClusterExtensionInstallRegistry(t *testing.T) {
 		assert.Equal(ct, &ocv1alpha1.BundleMetadata{Name: "prometheus-operator.2.0.0", Version: "2.0.0"}, clusterExtension.Status.ResolvedBundle)
 	}, pollDuration, pollInterval)
 
+	t.Log("By eventually reporting a successful unpacked")
+	require.EventuallyWithT(t, func(ct *assert.CollectT) {
+		assert.NoError(ct, c.Get(context.Background(), types.NamespacedName{Name: clusterExtension.Name}, clusterExtension))
+		cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, rukpakv1alpha2.TypeUnpacked)
+		if !assert.NotNil(ct, cond) {
+			return
+		}
+		assert.Equal(ct, metav1.ConditionTrue, cond.Status)
+		assert.Equal(ct, rukpakv1alpha2.ReasonUnpackSuccessful, cond.Reason)
+		assert.Contains(ct, cond.Message, "Successfully unpacked")
```

Comment:
Where did `"Successfully unpacked"` come from? I do not see it in operator-controller codebase. I would suggest only assert on strings which come from the code we control.

Reply:
This is a case where we're pulling a rukpak API value that should only be used in BundleDeployment. We'll need to update the reason and message checks.
OWNER: @bentito See PR #874